<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindowPopUpForm</class>
 <widget class="QWidget" name="LoginWindowPopUpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>211</width>
     <height>271</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="cLoginTab">
    <attribute name="title">
     <string>買家登入</string>
    </attribute>
    <widget class="QLineEdit" name="c_id">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>50</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="c_password">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>120</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cAcc">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帳號(ID):</string>
     </property>
    </widget>
    <widget class="QLabel" name="cPass">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>90</y>
       <width>96</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密碼(Password):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="c_login_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="c_signup_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="c_error_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>145</y>
       <width>138</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="sLoginTab">
    <attribute name="title">
     <string>賣家登入</string>
    </attribute>
    <widget class="QLineEdit" name="s_id">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>50</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="s_password">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>120</y>
       <width>137</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sPass">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>90</y>
       <width>96</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>密碼(Password):</string>
     </property>
    </widget>
    <widget class="QLabel" name="sAcc">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>86</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>帳號(ID):</string>
     </property>
    </widget>
    <widget class="QPushButton" name="s_login_button">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Log In</string>
     </property>
    </widget>
    <widget class="QPushButton" name="s_signup_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>90</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Sign Up</string>
     </property>
    </widget>
    <widget class="QLabel" name="s_error_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>145</y>
       <width>138</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
